content/posts/blog/Return_of_experience_on_XMPP_meetup_in_Krakow.md

```diff
@@ -0,0 +1,31 @@
+In May of 2017 a meetup was organized in Krakow, Poland. We announced it on the [XSF blog](https://xmpp.org/2017/04/first-krakow-xmpp-meetup/), and [on meetup.com](htpps://www.meetup.com/XMPP-Jabber-Messaging-Krakow/events/239508291/) (also a few social networks).  The reponse from the XMPP/Jabber community was very positive, as we were fresh off the heels of FOSDEM and many members had just recently met at the XMPP Summit.
+
+## Philosophy & goals
+
+Firstly, it was a cooperation between Tigase and MongooseIM. Both organizations provide open source XMPP server software and bring professional expertise on the protocol.  However, we wanted it to be a humble and fast organisation, rather than a meticulously planned event. Calling to mind the addage "done is better than perfect".  The ultimate goal of the meetup was to meet colleagues of the XMPP and Jabber community, brainstorm, and overall improve communication between users and developers of the protocol.  An additional benefit from the meeting so soon after the Summit was to strengthen the trend and popularity of XMPP meetups in the European Union.
+
+## Speakers and presentations
+
+Two speakers from Erlang Solutions and one from Tigase were on hand to provide presentations.
+
+Andrzej Wojcik of Tigase gave a presentation on "Why use XMPP for IoT?". A factual comparison: efficient and straight to the point as why other options do not provide a proper solution.  The alternatives, such as HTTP, AMQP, MQTT all have their advantages, but none bring a global solution as exhaustive as XMPP.  A demonstration was shown of a proof-of-concept of IoT devices being controlled VIA XMPP commands.
```

Comment:
nit: "on 'Why use XMPP for IoT?'" sounds on; maybe replace "on" with "titled"? Again, might just be me.

> as why other

"as to why other" maybe? This whole stentence feels a bit oddly structured. Maybe: "It was a factual comparison, efficient and straight to the point, as to why other options do not provide a proper solution."

nit: "proper solution" also sounds vague, what *is* a "proper solution"? Probably fine, but maybe there's a better word. Complete solution?

> as exhaustive as XMPP.

"as exhaustive as XMPP's [solution]"

> VIA XMPP

"via XMPP"